app/src/main/java/org/projectbuendia/client/ui/chart/ChartRenderer.java

```diff
@@ -168,7 +167,18 @@ void addObservations(List<Obs> observations) {
             }
         }
 
-        /** Returns a column for a given date. Creates a new column, if it does not exist yet */
+        /** Ensures that columns are shown for any days in which an order is prescribed. */
+        void addOrders(List<Order> orders) {
+            for (Order order : orders) {
+                if (order.stop != null) {
+                    for (DateTime dt = order.start; !dt.isAfter(order.stop.plusDays(1)); dt = dt.plusDays(1)) {
```

Comment:
@zestyping I added the order.stop.plusDays(1) to make the STOP day appear for a future order. Is that ok? If so it will be ready to merge!